addons/account_bank_statement_import/account_bank_statement_import.py

```diff
@@ -259,14 +289,19 @@ def _create_bank_statements(self, cr, uid, stmts_vals, context=None):
         num_ignored = len(ignored_statement_lines_import_ids)
         if num_ignored > 0:
             notifications += [{
-                'type': 'warning',
-                'message': _("%d transactions had already been imported and were ignored.") % num_ignored if num_ignored > 1 else _("1 transaction had already been imported and was ignored."),
+                'type': 'UserError',
```

Comment:
we must keep the type as ```warning``` because it's used in the js to determine the notification style.